packages/core/src/codewhispererChat/controllers/chat/telemetryHelper.ts

```diff
@@ -258,6 +260,7 @@ export class CWCTelemetryHelper {
                         acceptedLineCount: event.cwsprChatAcceptedNumberOfLines,
                         acceptedSnippetHasReference: false,
                         hasProjectLevelContext: this.responseWithProjectContext.get(event.cwsprChatMessageId),
+                        customizationArn: undefinedIfEmpty(getSelectedCustomization().arn),
```

Comment:
Does this correctly handle the case where:
1. user selects a customization X
2. user makes a chat request and gets a response back using customization X 
3. user selects a new customization Y
4. user interacts with message (e.g. copyToClipboard) on the last message that was received while customization X was active

If I understand the code correctly, we will record customization Y where we should actually record customizationX in this case

Reply:
you're right Ege, it won't handle the edge case you mentioned